README.md

```diff
@@ -1,2 +1,69 @@
-# Hello-world
-Set up a "Hello Microverse" project
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse, setting up "Hello Microverse" project for week 1.
+
+## Built With
+
+- Major languages : HTML & CSS
+- Frameworks : none
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project
+locally.** **Modify this file to match your project, remove sections that don't
+apply. For example: delete the testing section if the currect project doesn't
+require testing.**
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+### Setup
+
+### Install
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+## Authors
```

Comment:
- [OPTIONAL] Kindly remove unnecessary/empty sections. and I think it would be better to customize `Getting Started` section to explain to others about how to download and use your project. 👌

> For Example 💡
To get a local copy run `git@github.com:mrbhi/Hello-world.git`
Open `index.html` on any browser